src/main/java/uk/ac/ebi/eva/submission/entity/SubmissionProcessing.java

```diff
@@ -0,0 +1,106 @@
+    package uk.ac.ebi.eva.submission.entity;
+
+import org.springframework.lang.NonNull;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(schema = "eva_submissions", name = "submission_processing_status")
+public class SubmissionProcessing {
+
+    public SubmissionProcessing() {
+
+    }
+
+    public SubmissionProcessing(String submissionId) {
+        this.submissionId = submissionId;
+    }
+
+    @Id
+    @NonNull
+    @Column(nullable = false, name = "submission_id")
+    private String submissionId;
```

Comment:
```suggestion
    @Id
    @NonNull
    @Column(nullable = false, name = "submission_id")
    private String submissionId;
    
     @ManyToOne
    @JoinColumn(name = "submission_id", referencedColumnName = "submission_id", insertable = false, updatable = false)
    private Submission submission;
```

I think we should make the submissionId a foreign key referencing the submissionId from submission table